cmd/argoexec/commands/artifact/delete.go

```diff
@@ -2,15 +2,15 @@ package artifact
 
 import (
 	"context"
-	"encoding/json"
+	//"encoding/json"
 	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/spf13/cobra"
 
-	wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
-	executor "github.com/argoproj/argo-workflows/v3/workflow/artifacts"
+	//wfv1 "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
```

Comment:
clean up pls

Reply:
yeah, it's still in Draft state - not really ready for review yet